CONTRIBUTING.md

```diff
@@ -56,31 +56,31 @@ Some notes on GitHub PRs:
 ## Release Process (for Admin only)
 
 The release tags of this package follow exactly the MeiliSearch versions.<br>
-It means the `v0.17.0` tag in this repository corresponds to the DO image running with MeiliSerach `v0.17.0`.
+It means that, for example, the `v0.17.0` tag in this repository corresponds to the DO image running MeiliSerach `v0.17.0`.
 
 This repository currently does not provide any automated way to test and release the DO image.<br>
 **Please, follow carefully the steps in the next sections before any release.**
 
 ### Test before Releasing
 
-1. In (`scripts/deploy.sh`)[/scripts/deploy.sh] and [`server/tools/build-image.py`](/server/tools/build-image.py), change all the `vX.X.X` by the new MeiliSearch version. If you want to test with a MeiliSearch RC, replace them by the right RC version tag (`vX.X.XrcX`).
+1. In [`scripts/deploy.sh`](/scripts/deploy.sh) and [`server/tools/build-image.py`](/server/tools/build-image.py), change all the `vX.X.X` by the new MeiliSearch version. If you want to test with a MeiliSearch RC, replace them by the right RC version tag (`vX.X.XrcX`).
 
-2. Commit your changes on a new branch and open the related PR.
+2. Commit your changes on a new branch.
 
-3. Create a tag on the PR branch corresponding to the MeiliSearch version you've just changed(`vX.X.X` or `vX.X.XrcX`):
+3. Create a tag on the last commit of the branch corresponding to the MeiliSearch version you've just changed(`vX.X.X` or `vX.X.XrcX`):
 ```bash
 $ git tag vX.X.X
 $ git push origin <branch-name>
 ```
 
 4. Build the image:
 ```bash
-$ WIP
+$ python3 /server/tools/build-image.py
 ```
 
-3. ...
+This command will create a DigitalOcean Droplet on MeiliSearch's account and configure it in order to prepare the Marketplace image. It will then create a Snapshot, which should be ready to be published to the Marketplace. The Droplet will automatically be removed from the account after the image creation. The image name will be MeiliSearch-v.X.X.X-Debian-X.
```

Comment:
```suggestion
This command will create a DigitalOcean Droplet on MeiliSearch's account and configure it in order to prepare the Marketplace image. It will then create a snapshot, which should be ready to be published to the Marketplace. The Droplet will automatically be removed from the account after the image creation.<br>
The image name will be MeiliSearch-v.X.X.X-Debian-X.
```